woocommerce-services.php

```diff
@@ -1801,6 +1821,20 @@ public function are_woo_shipping_and_woo_tax_active() {
 		public function display_woo_shipping_and_woo_tax_are_active_notice() {
 			echo '<div class="error"><p><strong>' . esc_html__( 'Woo Shipping and Woo Tax plugins are already active. Please deactivate WooCommerce Shipping & Tax.', 'woocommerce-services' ) . '</strong></p></div>';
 		}
+
+		public function display_wcst_to_wcshipping_migration_notice() {
+			echo wp_kses_post(
+				'<div class="notice notice-error is-dismissible wcst-wcshipping-migration-notice"><p style="margin-bottom:0px">' .
+				 sprintf(
+					/* translators: %s: documentation URL */
+					__( 'WooCommerce Shipping & Tax is splitting into two dedicated extensions: WooCommerce Shipping and WooCommerce Tax. To minimize disruption, your settings and data will be carried over to the new extensions when you upgrade. <a href="%s" style="color:#3858E9;">Learn more about this change.</a>', 'woocommerce-services' ),
+					'https://wordpress.org/plugins/woocommerce-services/'
+				)  .
+				'</p>
+				<button style="color:#3858E9;margin: 12px 0;border: 1px solid #3858E9;padding: 11.5px 12px 11.5px 12px;border-radius: 2px;background-color:#fff;">Confirm update</button>
```

Comment:
`woocommerce-services/assets/stylesheets/style.scss` isn't loaded on the orders page. We could add some hooks to load a separate stylesheet for this notice, but I chose to use inline styles instead because this notice is a standalone component and its style is not reusable by others yet. If needed, we can always refactor afterwards.